hydra-node/test/Hydra/API/ServerSpec.hs

```diff
@@ -58,6 +58,37 @@ spec = parallel $ do
               failAfter 1 $ atomically (replicateM 2 (readTQueue queue)) `shouldReturn` [arbitraryMsg, arbitraryMsg]
               failAfter 1 $ atomically (tryReadTQueue queue) `shouldReturn` Nothing
 
+  it "sends all sendOutput history to all connected clients after a restart" $ do
+    queue1 <- atomically newTQueue
+    queue2 <- atomically newTQueue
```

Comment:
Can you move this queue initialization closer to where it actually is used? It's a bit out of context her.